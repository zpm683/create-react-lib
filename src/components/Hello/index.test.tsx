import { Hello } from ".";
import { render } from "@testing-library/react";

describe("Button unit test", () => {
  it("Snapshot", () => {
    const { container } = render(<Hello />);
    expect(container).toMatchSnapshot();
  });
});
