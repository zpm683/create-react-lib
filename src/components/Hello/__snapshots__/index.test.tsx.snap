// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button unit test Snapshot 1`] = `
<div>
  <p
    class="MuiTypography-root MuiTypography-body1"
  >
    Hello!
  </p>
</div>
`;
